packages/story-editor/src/components/keyboardShortcutsMenu/keyboardShortcutList.js

```diff
@@ -354,6 +354,16 @@ const shortcuts = {
             </kbd>
           ),
         },
+        {
+          label: __('Access floating toolbar', 'web-stories'),
+          shortcut: (
+            <kbd>
+              <kbd className="large-key">{prettifyShortcut('mod')}</kbd>
```

Comment:
Right now I have this shortcut as `ctrl+alt+p` instead of `mod+alt+p`. Happy to do whichever tho

Reply:
i'll change it here, i just replaced the kbd i copied poorly.